packages/manager/src/features/IAM/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Redirect, Route, Switch } from 'react-router-dom';
+
+import { ProductInformationBanner } from 'src/components/ProductInformationBanner/ProductInformationBanner';
+import { SuspenseLoader } from 'src/components/SuspenseLoader';
+
+import type { RouteComponentProps } from 'react-router-dom';
+
+const IAMLanding = React.lazy(() => import('./IAMLanding'));
+
+const UserDetails = React.lazy(() =>
+  import('./Users/UserDetailsLanding').then((module) => ({
+    default: module.UserDetailsLanding,
+  }))
+);
+
+type CombinedProps = RouteComponentProps;
+
+export const IdentityAccessManagement: React.FC<CombinedProps> = (props) => {
```

Comment:
We can express this in a cleaner way. We aren't combining any types to we don't need a type called `CombinedProps`. We also don't need to use `React.FC`. 

```suggestion
export const IdentityAccessManagement = (props: RouteComponentProps) => {
```

Reply:
done